tensorboard/webapp/notification_center/_redux/notification_center_selectors.ts

```diff
@@ -32,3 +32,10 @@ export const getNotifications = createSelector(
     return state.notifications;
   }
 );
+
+export const getLastReadTime = createSelector(
+  selectNotifications,
+  (state: NotificationState): number => {
+    return state.lastReadTimestamp ? state.lastReadTimestamp : -1;
```

Comment:
1. it should be `state.lastReadTimestamp ?? -1;` (otherwise lastReadTimestamp = 0 would return -1)
2. can you specify time unit in the state and the selector
3. I think this thing should return `null` to let the UI know that it is not loaded yet. WDYT? Not a strong opinion here.

Reply:
I think the UI only cares if the red dot should appear or not. If there is no timestamp in local storage, it is equivalent to `-1` which makes the condition (notification.dateInMs < lastReadTime) be true
I prefer to leave the logics on _redux side. The UI could be quite clean this way.
Unless... there are some cases that the `getLastReadTime` returns `null` for other reasons?

Reply:
1. addressed. Thanks!
2. Renamed `lastReadTimestamp` to `lastReadTimestampInMs`